tests/cli/test_run.py

```diff
@@ -246,3 +246,124 @@ def test_main_cannot_write_zuliprc_given_good_credentials(
         )
     )
     assert lines[-1] == expected_line
+
+
+@pytest.fixture
+def parameterized_zuliprc():
+    def func(zuliprc, config):
+        with open(zuliprc, "w") as f:
+            f.write("[api]\n\n")  # minimal to avoid Exception
+            f.write("[zterm]\n")
+            for key, value in config.items():
+                f.write("{key}={value}\n".format(
+                    key=key,
+                    value=value
+                ))
+        return zuliprc
+    return func
+
+
+@pytest.mark.parametrize('zulip_config', [{}])
+def test_successful_main_function_with_no_config(
+    parameterized_zuliprc, zulip_config, minimal_zuliprc,
+    capsys, mocker
+                                                ):
+    modified_zuliprc = parameterized_zuliprc(minimal_zuliprc, zulip_config)
+    mocker.patch('zulipterminal.core.Controller.__init__',
+                 side_effect=SystemExit)
+    with pytest.raises(SystemExit):
+        main(['-c', modified_zuliprc])
+    captured = capsys.readouterr()
+    lines = captured.out.strip().split("\n")
+    expected_lines = [
+        'Loading with:',
+        "   theme 'zt_dark' specified with no config.",
+        "   autohide setting 'no_autohide' specified with no config.",
+        "   maximum footlinks setting '3' specified with no config.",
+        "   color depth setting '256' specified with no config."
+                     ]
+    assert lines == expected_lines
+
+
+@pytest.mark.parametrize('zulip_config', [
+    {
+        "theme": "default",
+        "autohide": "autohide",
+        "footlinks": "disabled",
+        "notify": "enabled",
+        "color-depth": "256"
+    },
+    {
+        "theme": "default",
+        "autohide": "autohide",
+        "maximum-footlinks": "3",
+        "notify": "enabled",
+        "color-depth": "256"
+    }])
```

Comment:
These are the defaults, but given this PR I'd like to see a test for the max=0 and footlinks=enabled cases too. If the other values are the same, you can simplify the parametrization in any case and build the dict in the test, which would make the intent of this test clearer in any case :)